{"sections":[],"schemaVersion":{"minor":3,"major":0,"patch":0},"kind":"symbol","abstract":[{"type":"text","text":"A localized message describing how one might recover from the failure."}],"hierarchy":{"paths":[["doc:\/\/co.pointfree.VaporRouting\/documentation\/Vapor","doc:\/\/co.pointfree.VaporRouting\/documentation\/Vapor\/Abort","doc:\/\/co.pointfree.VaporRouting\/documentation\/Vapor\/Abort\/DebuggableError-Implementations"]]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/co.pointfree.VaporRouting\/documentation\/Vapor\/Abort\/recoverySuggestion-8rnsl"},"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"recoverySuggestion","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":"? { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["Linux"]}]}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/vapor\/abort\/recoverysuggestion-8rnsl"]}],"metadata":{"extendedModule":"Vapor","symbolKind":"property","role":"symbol","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"recoverySuggestion"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"roleHeading":"Instance Property","externalID":"s:5Vapor15DebuggableErrorPAAE18recoverySuggestionSSSgvp::SYNTHESIZED::s:5Vapor5AbortV","modules":[{"name":"Vapor"}],"title":"recoverySuggestion"},"references":{"doc://co.pointfree.VaporRouting/documentation/Vapor":{"type":"topic","url":"\/documentation\/vapor","kind":"symbol","title":"Vapor","abstract":[],"role":"collection","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/Vapor"},"doc://co.pointfree.VaporRouting/documentation/Vapor/Abort":{"fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"Abort","kind":"identifier"}],"abstract":[{"text":"Default implementation of ","type":"text"},{"code":"AbortError","type":"codeVoice"},{"text":". You can use this as a convenient method for throwing","type":"text"},{"text":" ","type":"text"},{"code":"AbortError","type":"codeVoice"},{"text":"s without having to conform your own error-type to ","type":"text"},{"code":"AbortError","type":"codeVoice"},{"text":".","type":"text"}],"url":"\/documentation\/vapor\/abort","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/Vapor\/Abort","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"Abort"}],"role":"symbol","title":"Abort","type":"topic"},"doc://co.pointfree.VaporRouting/documentation/Vapor/Abort/DebuggableError-Implementations":{"abstract":[],"url":"\/documentation\/vapor\/abort\/debuggableerror-implementations","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/Vapor\/Abort\/DebuggableError-Implementations","kind":"article","role":"collectionGroup","title":"DebuggableError Implementations","type":"topic"},"doc://co.pointfree.VaporRouting/documentation/Vapor/Abort/recoverySuggestion-8rnsl":{"type":"topic","url":"\/documentation\/vapor\/abort\/recoverysuggestion-8rnsl","abstract":[{"type":"text","text":"A localized message describing how one might recover from the failure."}],"kind":"symbol","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/Vapor\/Abort\/recoverySuggestion-8rnsl","role":"symbol","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"recoverySuggestion"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"kind":"text","text":"?"}],"title":"recoverySuggestion"}}}